tutorials/scripting/resources.rst

```diff
@@ -206,6 +206,8 @@ Let's see some examples.
     export(Resource) var sub_resource
     export(Array, String) var strings
 
+    # Make sure that every variable gets a default value. 
+    # Otherwise, there will be problems with creating and editing your resource via the inspector.
```

Comment:
```suggestion
    # Make sure that every parameter gets a default value. 
    # Otherwise, there will be problems with creating and editing
    # your resource via the inspector.
```